acra-censor/handlers/queryignore_handler.go

```diff
@@ -0,0 +1,44 @@
+package handlers
+
+type QueryIgnoreHandler struct {
+	ignoredQueries map[string]bool
+}
+
+func NewQueryIgnoreHandler() *QueryIgnoreHandler {
+	handler := &QueryIgnoreHandler{}
+	handler.ignoredQueries = make(map[string]bool)
+	return handler
+}
+
+func (handler *QueryIgnoreHandler) CheckQuery(query string) (bool, error) {
+
+	if handler.ignoredQueries[query]{
+		//do not continue query handling
+		return false, nil
+	}
+	return true, nil
+}
+
+func (handler *QueryIgnoreHandler) Reset() {
+	handler.ignoredQueries = nil
```

Comment:
`handler.ignoredQueries = make(map[string]bool)` will be better to avoid nil dereference